# RNCR3_Bioinformatic_code
This will contain all code used to process and analyze fastq files generated from RNCR3 Knockout NE4C cells


A trizol-chloroform extraction was used to isolate RNA from mouse embryonic fibroblasts, collected and expanded from day 12.5 embryos. 1Âµg of RNA was used for library preparation (Tecan Universal mRNA library Kit cat no.  0520). Samples were run on the Illumina NovaSEQ6000 system at a depth of 80million paired-end reads. Following de-multiplexing, fastQ files were assessed for quality using fastQC (version 0.11.8). Trimgalore (0.4.3) was used to remove adapters, jumping index content unique to Tecan library kit, and sections of reads with low quality with a minimum of 20bp read length. Samples were then aligned using tophat2 (version 2.1.1) utilizing bowtie2 (version 2.2.9), beyond default options for a sensitive paired end alignment, reads were aligned in reverse to accommodate the first-strand-sense reads unique to the Tecan library prep kit. Samples were then sorted and indexed using samtools (version 1.10). Read counts were assessed using htseq (version 0.9.1 under python distribution 3.6.3) and options were set for a paired-end reverse alignment. Differential expression was evaluated using deseq2 (version 3.14) within Rstudio (version 1.4.1103) utilizing R (version 4.1.2). Deseq2 analysis was
performed on all 12 samples simultaneously partioned by design based on transcript knockout length. Genes identified to be differentially expressed between wild-type and an experimental condition(multiple hypothesis testing p-value <0.05 and log2fold change >.5) were further submitted to a
likelihood ratio test within DEseq to eliminate batch effects as a possible contributor to differential
expression.
